packages/headers-parser/vitest.config.ts

```diff
@@ -1,7 +1,18 @@
+import type { BenchmarkUserOptions } from 'vitest'
 import { defineConfig } from 'vitest/config'
 
+const benchmarkOptions: BenchmarkUserOptions = {}
+// Tinybench dropped support for Node 14 recently so we need to skip benchmark tests for it
+// https://github.com/tinylibs/tinybench/pull/40 and
+// https://github.com/tinylibs/tinybench/pull/43
+// This is just a quick workaround to not run the bench tests in node 14 while we still support it
+if (process.version.startsWith('v14')) {
+  benchmarkOptions.exclude = ['tests/**/*.bench.ts']
+}
+
```

Comment:
With the lockfile update I've brought an update from tinybench from `2.5.0` to `2.5.1` which dropped support for node 14. We'll still run the benchmarks for the other CI test suites, but this skips it for node 14.